File: naive/463.sort-integers/Solution.java
public class Solution {
    /**
     * @param A: an integer array
     * @return: nothing
     */
    public void sortIntegers(int[] A) {
        // write your code here
        if(A==null || A.length<2){
            return;
        }
        for (int i=A.length-1;i>0;i--){
            for (int j=0; j<i; j++){
                if (A[j]>A[j+1]){
                    //swap
                    int tmp = A[j];
                    A[j]=A[j+1];
                    A[j+1] = tmp;
                }
            }
        }
    }
}

File: naive/463.sort-integers/Solution.go
package main

import "fmt"

func main() {
	A := &[]int{}
	fmt.Println(A)
	sortIntegers(A)

}

func sortIntegers(A *[]int) {
	// write your code here
	deA := *A
	if A == nil || len(deA) < 2 {
		return
	}
	//iterate over len of A
	for i := len(deA) - 1; i > 0; i-- {
		//iterate over elenements in a sub list
		for j := 0; j < i; j++ {
			//swap
			if deA[j] > deA[j+1] {
				deA[j], deA[j+1] = deA[j+1], deA[j]
			}
		}
	}
}
